src/app/dashboard/Missions/Missions.jsx

```diff
@@ -159,7 +159,7 @@ const mapStateToProps = (state, ownProps) => {
   let inPlanning = Mission.selectInPlanning(state);
   let inProgress = Mission.selectInProgress(state);
   let inDone = Mission.selectInDone(state);
-  let volunteers = state.firestore.ordered.volunteers;
+  let volunteers = state.firestore?.ordered?.volunteers;
```

Comment:
this should be a selector, but I have been lazy

Reply:
I can add it as a new selector.

Reply:
yeah, I think its a good idea, 
User.selectVolunteers ?

Reply:
I added it to Mission model. Let me know if it's correct.